uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request.controller.js

```diff
@@ -191,20 +187,24 @@ define([
      * Initialize expiryDate on UI from server's toil_expiry_date
      */
     function initExpiryDate () {
-      if (vm.canManage) {
-        vm.uiOptions.expiryDate = vm._convertDateFormatFromServer(vm.request.toil_expiry_date);
+      if (parentCtrl.canManage) {
+        parentCtrl.uiOptions.expiryDate = parentCtrl.convertDateFormatFromServer(parentCtrl.request.toil_expiry_date);
       }
     }
 
     /**
      * Initialises watcher for accrue value
```

Comment:
again, this documentation is kinda pointless. Say what it happens when the accrue value changes

Reply:
Done.